chain/chunks/src/lib.rs

```diff
@@ -142,8 +142,8 @@ pub struct ShardsManager {
     requested_one_parts: RequestPool,
     requested_chunks: RequestPool,
 
-    requests_fifo: VecDeque<(ShardId, ChunkHash, PeerId, u64)>,
-    requests: HashMap<(ShardId, ChunkHash, u64), HashSet<(PeerId)>>,
+    requests_fifo: VecDeque<(ShardId, ChunkHash, CryptoHash, u64)>,
+    requests: HashMap<(ShardId, ChunkHash, u64), HashSet<(CryptoHash)>>,
```

Comment:
Why do you have parenthesis around `CryptoHash`?

Reply:
It was like this before I get my hands into this, but yes, I think this was for no good reason as I see in the code, will change it.